code/modules/admin/topic.dm

```diff
@@ -937,11 +937,11 @@
 						continue
 					joblist += jobPos
 			if("ghostroles")
-				joblist += list("pAI", "posibrain", "drone", "deathsquad", "lavaland")
+				joblist += list(ROLE_PAI, "posibrain", "drone", "deathsquad", "lavaland")

```

Comment:
Would probably be for the best to replace the rest of these with defines I would think.